ManageWikiExtensions.php

```diff
@@ -862,6 +862,14 @@
 		'requires' => [],
 		'section' => 'parserhooks',
 	],
+	'skinperpage' => [
+		'name' => 'SkinPerPage',
```

Comment:
"Skin per page" this should now match ExtensionRegistry name, once my ManageWiki and CreateWiki changes are done, so I'd prefer to do that now for this.

Reply:
Done.